spec/factories/actors.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :actor do
+  end
```

Comment:
I struggled with this - what "default" attributes of an actor/movie/director/character would you add here?  All the attributes are specific to each record, so I'm not sure what default attributes would be useful.